ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -92,6 +92,7 @@ descriptor:
   condensate: "Liquid that acculates on surfaces as a byproduct of condensation (i.e., water vapor condensing into liquid)"
   condensing: "The process of a gas (or vapor) changing to liquid."
   conditioning: "Treatment of a medium."
+  conditioned: "A substance modified to meet specific temperature requirements"
```

Comment:
Please make this more general, similar to conditioning - maybe just take out "temperature"

Reply:
Updated